### PR TITLE
HPCC-30890 Changelog automation to build-assets workflow

### PR DESCRIPTION
**Pull request by Michael-Gardner**
## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [x] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Comment by Michael-Gardner** [comment]
@cloLN and @xwang2713 please give this a good look before I ping Gordon on it.

I tested all the logic with a standalone workflow on my own repository.

The output: https://github.com/Michael-Gardner/HPCC-Platform/releases/tag/community_9.10.0-rc1
Action run: https://github.com/Michael-Gardner/HPCC-Platform/actions/runs/11392119417

**Review by xwang2713 — COMMENTED** [review]
Upload part looks good

**Comment by cloLN** [comment]
Output files look good.

**Review by GordonSmith — CHANGES_REQUESTED** [review]
Do we want to relocate `gfortil/PrettyGitLogs` into hpcc-systems now or in the future?

File: .github/workflows/build-assets.yml [hunk 1]
```diff
@@ -695,3 +699,111 @@ jobs:
...
+          repository: ${{ github.repository_owner }}/ECLIDE
+          ref: ${{ needs.preamble.outputs.eclide_ref }}
+          submodules: recursive
+          path: ${{ needs.preamble.outputs.folder_eclide }}
+          fetch-tags: true
+          fetch-depth: 0
+      - name: Checkout PrettyGitLogs
+        uses: actions/checkout@v4
+        with:
+          repository: hpcc-systems/PrettyGitLogs
```
Comment (Michael-Gardner) [inline .github/workflows/build-assets.yml:739]: Fixed so we use the PrettyGitLogs in the hpcc-systems namespace

**Review by xwang2713 — APPROVED** [review]
Looks good

**Review by GordonSmith — APPROVED** [review]

**Pull request merged**